### ISSUE TITLE
Getting `rest` NameError exception

### ISSUE THREAD

**Issue by booyaa** [brrzap/tootstream#1]
```
[@booyaa (default)]: t fucking nerds lol... slack client for the c64 https://github.com/jeff-1amstudios/c64-slack-client
Traceback (most recent call last):
  File "/home/pi/tootstream/tootstream/lib/python3.4/site-packages/click_shell/core.py", line 36, in invoke_
    parent=self.ctx)
  File "/home/pi/tootstream/tootstream/lib/python3.4/site-packages/click-6.6-py3.4.egg/click/core.py", line 696, in main
    rv = self.invoke(ctx)
  File "/home/pi/tootstream/tootstream/lib/python3.4/site-packages/click-6.6-py3.4.egg/click/core.py", line 889, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/pi/tootstream/tootstream/lib/python3.4/site-packages/click-6.6-py3.4.egg/click/core.py", line 534, in invoke
    return callback(*args, **kwargs)
  File "./tootstream.py", line 157, in toot
    cprint(rest, fg('magenta') + attr('bold') + attr('underlined'))
NameError: name 'rest' is not defined
```

packages installed (reusing the virtualenv for the stable version of tootstream)

```
appdirs (1.4.3)
click (6.6)
click-shell (1.0)
colored (1.3.4)
dateutils (0.6.6)
Mastodon.py (1.0.6)
packaging (16.8)
pip (9.0.1)
pyparsing (2.2.0)
python-dateutil (2.6.0)
pytz (2016.7)
readline (6.2.4.1)
requests (2.12.3)
setuptools (34.3.3)
six (1.10.0)
termcolor (1.1.0)
tootstream (0.1)
wheel (0.29.0)
```

using the following commit of the experimental branch: ea337cb540bffb8adffcd566384fe99fd30cfa61

I did cloned your repo, installed click-shell as per [this](https://github.com/magicalraccoon/tootstream/issues/60#issuecomment-294344428) issue.

**Comment by brrzap** [comment]
sorry!  merges keep trying to put those back in.  can't tell you how many times i've changed *[that exact line](https://github.com/brrzap/tootstream/blame/ea337cb540bffb8adffcd566384fe99fd30cfa61/tootstream.py#L157)*.  fixed in 1ef774885a309ddf2a616230bc7d87b2b643fccc and i'll try to remember to `grep` before commit after merges...  thanks!

**Comment by booyaa** [comment]
yup that fixed it! thank you!
